Makefile

```diff
@@ -31,7 +31,8 @@ DATA_built  = $(RELEASE_SQL) \
 
 DOCS        = README.pg_sphere COPYRIGHT.pg_sphere
 REGRESS     = init tables points euler circle line ellipse poly path box index \
-              contains_ops contains_ops_compat bounding_box_gist gnomo epochprop
+              contains_ops contains_ops_compat bounding_box_gist gnomo epochprop\
```

Comment:
There should be a space between `echoprop` and the backslash on this line.

Reply:
resolved